test/staleIfError.js

```diff
@@ -77,10 +76,16 @@ describe('Stale-If-Error', () => {
   it('throws the error that starting the cache throws', async () => {
     api.get('/').thrice().reply(200, defaultResponse.body, defaultHeaders);
     const cache = createCache();
-    const startError = new Error('Error starting da cache');
+    const expectedErrorMessage = 'Error starting da cache';
+    const startError = new Error(expectedErrorMessage);
     sandbox.stub(cache, 'start').rejects(startError);
-
-    return rejects(() => requestWithCache(cache, { ignoreCacheErrors: false }), startError);
+    try {
+      await requestWithCache(cache, {
+        ignoreCacheErrors: false
+      });
+    } catch (error) {
+      assert.equal(error.message, expectedErrorMessage);
+    }
```

Comment:
Need to fail test in case we don't get into the `catch`:
```
} catch (error) {
  assert.equal(error.message, expectedErrorMessage);
  return ;
}
assert.fail('No error thrown');
```

Reply:
done